metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -309,13 +309,13 @@ def get_urns_by_filter(
     filter_criteria = []
     if env:
         filter_criteria.append({"field": "origin", "value": env, "condition": "EQUAL"})
-
+    entity_type_lower = entity_type.lower()
     if (
         platform is not None
-        and entity_type == "dataset"
-        or entity_type == "dataflow"
-        or entity_type == "datajob"
-        or entity_type == "container"
+        and entity_type_lower == "dataset"
```

Comment:
Is there anywhere else we do these types of entity type comparisons in the CLI?

Reply:
There are a few others, but they look to be already using .lower() or are in tests